services/QuillLMS/app/controllers/api/v1/app_settings_controller.rb

```diff
@@ -0,0 +1,17 @@
+class API::V1::AppSettingsController < ApplicationController
+  def index
+    app_settings = AppSetting.all
+    user = User.find app_setting_index_params
+
+    jayson = app_settings.each_with_object({}) do |app_setting, memo| 
+      memo[app_setting.id] = app_setting._enabled?(user) 
```

Comment:
Shouldn't this use the setting `name` as the key?

Reply:
Per above conversations, this API is subject to change. But currently, `_enabled?` does not require a name, because the AppSetting instance already has access to itself.

Reply:
Will use name here.